### PR TITLE
Create example for testing parallel ECS updates

### PR DESCRIPTION
**Pull request by flostadler**
Adding an example to test parallel updates to components like ECS services. Currently component updates are serialized (#1250), but https://github.com/pulumi/pulumi/issues/7629 fixed that.

I used this test for verifying the changes locally, now that the change is released we can use this as a regression test

### CODE REVIEW

File: examples/examples_nodejs_test.go [hunk 1]
```diff
@@ -201,6 +202,24 @@ func TestVpcMultipleSimilarSubnetSpecArgs(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAccEcsParallel(t *testing.T) {
+	maxDuration(15*time.Minute, t, func(t *testing.T) {
```
Comment (flostadler) [inline examples/examples_nodejs_test.go:206]: Deploying and updating 10 of these ECS services took 1+ hours before. Now that it's all parallel it runs in under 10min

**Review by corymhall — APPROVED** [review]
Looks good! My only question is whether we need this test in `pulumi-awsx`. I feel like a regression test belongs in `pu/pu` since that is where the breaking change would happen and where the fix would be needed.

**Comment by flostadler** [comment]
> Looks good! My only question is whether we need this test in `pulumi-awsx`. I feel like a regression test belongs in `pu/pu` since that is where the breaking change would happen and where the fix would be needed.

Generally I agree with you, but this aspect of pu/pu is under-tested right now. I tried adding some tests for this area (provider server) there, but this would've required to refactor the provider server, which would be quite the rabbit hole.

**Pull request merged**